### ISSUE TITLE
rustdesk module

### ISSUE THREAD

**Issue by bpmct** [coder/registry#79]
Would probably require a [rustdesk](https://rustdesk.com/) server.

Please check #213 for the e contribution flow.

**Comment by DevelopmentCats** [comment]
@bpmct Are you more or less wanting a module to install a rustdesk agent in the workspace, so that it can be connected to from the rustdesk server?

**Comment by bpmct** [comment]
Yep!

**Comment by bpmct** [comment]
I think they have an HTML5 client too, so we should install and expose that via a `coder_app`

**Comment by DevelopmentCats** [comment]
From what I recall their Web client is only available to enterprise user's at cost, but I will check and make sure that this hasn't changed.

I was recently looking into rustdesk as a personal remote management solution, and I remember having trouble with that aspect of it

**Comment by BenraouaneSoufiane** [comment]
Hello, can I work on this?

**Comment by bpmct** [comment]
@BenraouaneSoufiane we currently use Algora for payments and support these methods: https://algora.io/docs/payments

If this doesn't work for you, let me know

**Comment by BenraouaneSoufiane** [comment]
> [@BenraouaneSoufiane](https://github.com/BenraouaneSoufiane) we currently use Algora for payments and support these methods: https://algora.io/docs/payments
> 
> If this doesn't work for you, let me know

Thank you for your reply, Stripe not support my country

**Comment by BenraouaneSoufiane** [comment]
> [@BenraouaneSoufiane](https://github.com/BenraouaneSoufiane) Which country? According to the docs linked above, Algeria is supported which is the country listed in your profile
Yes, it's supported, now I've setup my account (sorry for frustrating)

**Comment by BenraouaneSoufiane** [comment]
@matifali if possible can you increase the amount to be for $400 or 500$ because it was just a shell script that I install such a package & another tf script file

**Comment by bpmct** [comment]
Hi @BenraouaneSoufiane - it doesn't seem like the PR you submitted does anything practical for the user.

I understand there was more complexity than expected, but based on the quality of the initial PR and your immediate request to release the funds before the PR was approved I don't think it makes sense for us to continue working with you on this. Even if we were to raise it to $400/500, I don't have confidence in the ability for this problem to be solved in an elegant way, meeting the needs of our users.

As mentioned in the bounty guidelines

> If anything is unclear, ask for clarification before starting as this will help avoid potential rework

We'll be giving you a partial payout for your time, but we are not interested in working with you further on this and will close this issue until we will write clearer requirements.
